Gems/ROS2/Code/Source/VehicleDynamics/WheelDynamicsData.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+namespace VehicleDynamics
+{
+    //! Data structure to pass wheel dynamics data for a single wheel entity.
+    struct WheelDynamicsData
+    {
+        AZ::EntityId m_wheelEntity; //!< An entity which is expected to have a WheelControllerComponent.
+        AZ::Vector3 m_driveAxis; //!< An axis of force application for the wheel to move forward.
+        float m_wheelRadius; //!< Radius of the wheel in meters.
+    };
+
+    //! Data structure to pass steering dynamics data for a single steering entity.
+    struct SteeringDynamicsData
+    {
+        AZ::EntityId m_steeringEntity; //!< Steering entity needs to be connected (directly or indirectly) by a Joint with a wheelEntity.
+        AZ::Vector3 m_turnAxis; //!< An axis of force application for the steering element to turn the attached wheel sideways.
```

Comment:
repeat: Should `m_turnAxis` be initialized to a zero vector.
```suggestion
        AZ::Vector3 m_turnAxis{}; //!< An axis of force application for the steering element to turn the attached wheel sideways.
```